src/Storage/Storage.Management/help/New-AzRmStorageContainer.md

```diff
@@ -31,20 +31,27 @@ The **New-AzRmStorageContainer** cmdlet creates a Storage blob container
 ## EXAMPLES
 
 ### Example 1: Create a Storage blob container with Storage account name and container name, with metadata
-```
+```powershell
 PS C:\>New-AzRmStorageContainer -ResourceGroupName "myResourceGroup" -AccountName "myStorageAccount" -ContainerName "myContainer" -Metadata @{tag0="value0";tag1="value1";tag2="value2"}
 ```
 
 This command creates a Storage blob container with Storage account name and container name, with metadata.
 
 ### Example 2: Create a Storage blob container with Storage account object and container name, with public access as Blob
-```
+```powershell
 PS C:\>$accountObject = Get-AzStorageAccount -ResourceGroupName "myResourceGroup" -AccountName "myStorageAccount"
 PS C:\>New-AzRmStorageContainer -StorageAccount $accountObject -ContainerName "myContainer" -PublicAccess Blob
 ```
 
 This command creates a Storage blob container with Storage account object and container name, with public access as Blob.
 
+### Example 3
```

Comment:
This can be covered by example #1

Reply:
This example doesn't have the Metadata option, doesn't that make it a new example?